src/dba/models/Hashlist.class.php

```diff
@@ -197,4 +208,5 @@ function setBrainFeatures($brainFeatures) {
   const NOTES = "notes";
   const BRAIN_ID = "brainId";
   const BRAIN_FEATURES = "brainFeatures";
+  const CREATOR = "creatorId";
```

Comment:
should be `const CREATOR_ID = "creatorId";`

Reply:
Fixed in https://github.com/hashtopolis/server/pull/778/commits/31c8787c8a5766d6ff0a9c989c4ae9b99037e554